packages/playground/react/__tests__/react.spec.ts

```diff
@@ -16,3 +16,20 @@ test('should hmr', async () => {
   // preserve state
```

Comment:
Why does this test pass when editFile doesn't write to the file when `isBuild` is `true`?